engines/bops_admin/app/controllers/bops_admin/application_types_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module BopsAdmin
+  class ApplicationTypesController < ApplicationController
+    before_action :set_application_types, only: %i[index]
+    before_action :set_application_type, only: %i[show]
+
+    def index
+      respond_to do |format|
+        format.html
+      end
+    end
+
+    def show
+      respond_to do |format|
+        format.html
+      end
+    end
+
+    private
+
+    def set_application_types
+      @application_types = current_local_authority.application_types.by_name
+    end
+
+    def set_application_type
+      @application_type = current_local_authority.application_types.find(application_type_id)
+    end
```

Comment:
If these hooks are each just being used once I think we could just inline them.
```suggestion
    before_action :set_application_type, only: %i[show]

    def index
      @application_types = current_local_authority.application_types.by_name

      respond_to do |format|
        format.html
      end
    end

    def show
      @application_type = current_local_authority.application_types.find(application_type_id)

      respond_to do |format|
        format.html
      end
    end

    private
```